staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -62,9 +62,14 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 // the method lazily initializes a generic plugin that is appended to the list of pluginInitializers
 // note this method uses:
 //  genericconfig.LoopbackClientConfig
-//  genericconfig.SharedInformerFactory
 //  genericconfig.Authorizer
-func (a *AdmissionOptions) ApplyTo(c *server.Config, informers informers.SharedInformerFactory, pluginInitializers ...admission.PluginInitializer) error {
+func (a *AdmissionOptions) ApplyTo(
+	c *server.Config,
+	informers informers.SharedInformerFactory,
+	proxyClientCert []byte,
```

Comment:
this name isn't correct.  It's a `serverIdentifyingClientCert`.  Has nothing to do with any proxy.

Reply:
sorry, shamelessly adopted the name from the existing code :)

Reply:
> sorry, shamelessly adopted the name from the existing code :)

np.  I commented about it being the wrong cert/key pair.  I don't think it will block this pull (pre-existing), but its quite wrong.